test/torchaudio_unittest/io/playback_test.py

```diff
@@ -0,0 +1,65 @@
+from unittest.mock import patch
+
+import torch
+from parameterized import parameterized
+from torchaudio.io import play_audio, StreamWriter
+from torchaudio_unittest.common_utils import get_sinusoid, skipIfNoDevice, skipIfNoMacOS, TorchaudioTestCase
+
+
+@skipIfNoDevice
+@skipIfNoMacOS
+class PlaybackInterfaceTest(TorchaudioTestCase):
+    @parameterized.expand(
+        [(torch.uint8,), (torch.int16,), (torch.int32,), (torch.int64,), (torch.float32,), (torch.float64,)]
```

Comment:
Please use str-type, as in some cases, this arguments become a part of test name and path, and cause an issue.

```suggestion
        [("uint8",), ("int16",), ("int32",), ("int64",), ("float32",), ("float64",)]
```

then inside of the test, do `dtype = getattr(torch, dtype)`